eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -48,15 +51,73 @@ steps:
   env:
     GH_TOKEN: $(azuresdk-github-pat)
 
+- task: PowerShell@2
+  displayName: 'Update Docs.MS CI Targeted Packages'
+  condition: and(succeededOrFailed(), eq('${{ parameters.OnboardingBranch }}',''))
+  inputs:
+    targetType: filePath
+    filePath: ${{ parameters.ScriptDirectory }}/update-docs-ci.ps1
+    arguments: >
+      -ArtifactLocation ${{ parameters.ArtifactLocation }}
+      -WorkDirectory "${{ parameters.WorkingDirectory }}"
+      -RepoId ${{ parameters.RepoId }}
+      -Repository ${{ parameters.PackageRepository }}
+      -ReleaseSHA ${{ parameters.ReleaseSha }}
+      -CIRepository "${{ parameters.WorkingDirectory }}/repo"
+      -Configs "${{ parameters.CIConfigs }}"
+    pwsh: true
+  env:
+    GH_TOKEN: $(azuresdk-github-pat)
+
 - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
   parameters:
     RepoName: ${{ parameters.TargetDocRepoName }}
     RepoOwner: ${{ parameters.TargetDocRepoOwner }}
     PRBranchName: ${{ parameters.PRBranchName }}
-    CommitMsg: "Update readme content for ${{ parameters.ArtifactName }}"
-    PRTitle: "Docs.MS Readme Update."
-    BaseBranchName: smoke-test
+    CommitMsg: "Update docs metadata and targeting for release of ${{ parameters.ArtifactName }}"
+    PRTitle: "Docs.MS Release Updates for ${{ parameters.ArtifactName }}"
+    BaseBranchName: ${{ parameters.SourceBranchName }}
     WorkingDirectory: ${{ parameters.WorkingDirectory }}/repo
     ScriptDirectory: ${{ parameters.WorkingDirectory }}/${{ parameters.ScriptDirectory }}
     GHReviewersVariable: ${{ parameters.GHReviewersVariable }}
     GHTeamReviewersVariable: ${{ parameters.GHTeamReviewersVariable }} 
+
+- ${{if ne( parameters['OnboardingBranch'], '')}}:
+  - pwsh: |
+      Push-Location ${{ parameters.WorkingDirectory }}/repo
+
+      git checkout *
```

Comment:
might want to do `git reset --hard HEAD` if you want to be sure to clean-out any modifications as well as added or removed files.